resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -179,3 +179,66 @@ def test_cutadapt_amplicon(self):
                          compression='gzip')
         self.assertFiles(filtered_reads, 'fastq2', ['cutadapt_trimmed_mate2.fastq.gz'],
                          compression='gzip')
+
+    def test_bbduk_single(self):
+        inputs = {'src': ['bbduk_reads.fastq.gz']}
```

Comment:
Rename the test file to ''bbduk reads.fastq.gz" and fix the error related to the whitespace in the filename. I can help with refactoring.